### PR TITLE
Updates README and associated samples

### PR DESCRIPTION
**Pull request by jdwelch**
+ fixes broken links
+ minor copyedits to improve readability
+ remove '$' sigils for easier copypasta
+ remove superfluous k8s sample workflow
+ sets refreshTime in sample k8s workflow to a valid value

Signed-off-by: JD Welch <jdwelch@puppet.com>

### CODE REVIEW

File: k8s/vpc-workflow.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ metadata:
   name: vpc-workflow
 spec:
   workflowName: aws_vpc_yaml
-  refreshTime: 65
+  refreshTime: 60
```
Comment (jdwelch) [inline k8s/vpc-workflow.yaml]: The [spec](https://github.com/lyraproj/lyra/blob/master/k8s/lyra_v1alpha1_workflow_crd.yaml#L46) has a limit of '60':

File: k8s/vpc-workflow.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ metadata:
   name: vpc-workflow
```
Comment (jdwelch) [inline k8s/vpc-workflow.yaml]: Re: `apiVersion: lyra.example.com/v1alpha1`; we should change this to a real URI, probably.

File: README.md [hunk 3]
```diff
@@ -18,37 +18,36 @@ A Workflow can be packaged as a deployable unit and consumed from any of these i
...
+This Workflow manages several resources on AWS and incorporates external data (for Tags, loaded [here](workflows/aws_vpc_yaml.yaml#L6) using a new Golang implementation of [hiera](https://github.com/lyraproj/hiera)), as specified in the [data.yaml](data.yaml) file. Lyra will use AWS credentials as configured in `~/.aws/credentials`.
 
-For the examples using Terraform providers (e.g. `typespace=>'TerraformAws'`), region is currently hard-coded to `eu-west-1`. For non-Terraform providers (e.g. `typespace=>'aws'`), Lyra will use the default region supplied in your `~/.aws/config`. 
+> NB regarding regions: For the [examples](workflows/) using Terraform providers (e.g. `typespace=>'TerraformAws'`), region is currently hard-coded to `eu-west-1`. For non-Terraform providers (e.g. `typespace=>'aws'`), Lyra will use the default region supplied in your `~/.aws/config`.
 
 ### Deploying Workflows with Kubernetes
```
Comment (jdwelch) [inline README.md]: TODO: Add something about *why* you'd want to do this, how the behaviour is different, etc.

File: README.md [hunk 4]
```diff
@@ -67,7 +66,7 @@ Here’s a proposed roadmap for the project. Given the infancy of the project, i
 ### Language Support
 - [x] Puppet
 - [x] YAML
-- [ ] TypeScript - [**IN PROGRESS**](https://github.com/lyraproj/lyra/issues/42)
+- [X] TypeScript
```
Comment (jdwelch) [inline README.md]: TODO: TypeScript needs docs

File: k8s/vpc-workflow.yaml (hunk 1 above)
Comment (ahpook) [reply k8s/vpc-workflow.yaml]: good catch - @markfuller and i ran into this too, i think it was an accidental checkin.

**Pull request merged**